### PR TITLE
Updated settings (no css style I promise :3)

### PR DESCRIPTION
**Pull request by 21DP3EFrei**

### CODE REVIEW

File: resources/js/Pages/Settings/Role.vue [hunk 1]
```diff
@@ -18,139 +20,67 @@ function handleSave() {
...
-              <PrimaryButton>
-                Save Changes
-              </PrimaryButton>
-              <button @click="handleCancel" class="border border-white text-white px-4 py-2 rounded-full hover:bg-red-700 hover:border-red-700 hover:text-white focus:outline-none transition-colors duration-300 ease-in-out">
-                X
-              </button>
-            </div>
+        <!-- Save/Cancel Buttons -->
+        <div class="flex justify-end mb-6 space-x-4">
+          <PrimaryButton @click="handleSave" class="px-6">Save Changes</PrimaryButton>
```
Comment (M7PAX) [inline resources/js/Pages/Settings/Role.vue]: better use daisy ui button, just add extra style if its special button, like all caps

This:
<button **class="btn"**><button/>

File: resources/js/Pages/Settings/Role.vue [hunk 2]
```diff
@@ -18,139 +20,67 @@ function handleSave() {
...
-            </main>
-          </div>
+        <!-- Members Table Section -->
+        <div class="bg-gray-700 p-8 rounded-lg shadow-lg">
+          <h1 class="text-3xl text-white mb-6">Role Settings</h1>
+
+          <!-- Add Role Button -->
+          <PrimaryButton @click="AddRole" class="mb-6">
+            Add Role
+          </PrimaryButton>
```
Comment (M7PAX) [inline resources/js/Pages/Settings/Role.vue]: daisy ui (first comment)

File: resources/js/Pages/Settings/Server.vue [hunk 3]
```diff
@@ -18,183 +19,84 @@ function handleSave() {
...
-                      <input
-                        type="text"
-                        id="serverName"
-                        class="w-full mt-1 p-2 rounded bg-gray-700 text-white focus:outline-none focus:ring focus:ring-[#285aff]"
-                        placeholder="Enter your server name"
-                      />
-                    </div>
-                  </div>
+        <div class="flex justify-end mb-6 space-x-4">
+          <PrimaryButton @click="handleSave" class="px-6">Save Changes</PrimaryButton>
```
Comment (M7PAX) [inline resources/js/Pages/Settings/Server.vue]: daisy ui (first comment)

**Comment by Lempac** [comment]
[Settings](https://trello.com/c/LzsnmODg/23-settings)

**Review by M7PAX — CHANGES_REQUESTED** [review]

File: resources/js/Pages/Settings/Role.vue (hunk 1 above)
Comment (Lempac) [reply resources/js/Pages/Settings/Role.vue]: @M7PAX Better question why do we have does component at all(Primary button, Danger button)?

**Review by M7PAX — APPROVED** [review]
LGTM

**Review by Lempac — APPROVED** [review]
Adding

**Pull request merged**